packages/protocol/contracts/governance/Governance.sol

```diff
@@ -90,6 +97,8 @@ contract Governance is IGovernance, Ownable, Initializable, UsingLockedGold, Ree
   mapping(address => ContractConstitution) private constitution;
   mapping(uint256 => Proposals.Proposal) private proposals;
   mapping(address => Voter) public voters;
+  mapping(bytes32 => HotfixRecord) public hotfixes;
+  Proposals.Proposal private hotfixProposal;
```

Comment:
why is this a state variable? should be local to a function

Reply:
existing initializer in Proposals contract operates on a storage reference, must be entirely duplicated for memory reference

Reply:
I think it's worth reworking this so we can use `memory` rather than `storage`